cocos/core/assets/morph-rendering.ts

```diff
@@ -48,6 +49,7 @@ export class StdMorphRendering implements MorphRendering {
                 continue;
             }
 
+            const hasFeatureTextureFloat = gfxDevice.hasFeature(GFXFeature.TEXTURE_FLOAT);
```

Comment:
I don't understand, it's never used afterward

Reply:
Unused code. Has been removed: https://github.com/cocos-creator/engine/pull/6937/commits/9877e50da83a644e8b37f233f22a4bfeb9a4c212